llvm/lib/Analysis/TargetLibraryInfo.cpp

```diff
@@ -334,6 +335,7 @@ static void initializeLibCalls(TargetLibraryInfoImpl &TLI, const Triple &T,
     TLI.setUnavailable(LibFunc_frexpl);
     TLI.setUnavailable(LibFunc_hypotl);
     TLI.setUnavailable(LibFunc_ldexpl);
+    TLI.setUnavailable(LibFunc_lgammal);
```

Comment:
As a pre-commit can you add new baseline TLI checker tests for the windows and linux cases? This is untested, and this is a problem in every one of these new libcall patches.

Reply:
Here is the PR: https://github.com/llvm/llvm-project/pull/114556